alert.css

```diff
@@ -0,0 +1,97 @@
+/* container for alert */
+.alert-container{
+    display:flex;
+    width:800px;
+    flex-direction:column;
+    gap:40px;
+    padding:40px 40px;
+    flex-wrap:wrap;
+    width:800px;
+    background-color:black;
+  }
+  
+  /* alert-box container */
+  .alert-box{
+    display:flex;
+    width:400px;
+    height:70px;
+    background-color:rgb(255 244 244);
```

Comment:
Instead of using the hardcoded values for color, you can define the CSS variable.

Reply:
ok ! I will fix it !